### PR TITLE
More changes for 1.5

### PR DESCRIPTION
**Pull request by humblearner**

### CODE REVIEW

File: docs/source/actions.rst [hunk 1]
```diff
@@ -110,9 +110,6 @@ Currently the system provides the following runners:
    information, please refer to the :doc:`Workflows </workflows>` and
    :doc:`CloudSlang </cloudslang>` sections of the documentation.
 
-   Note: This runner is currently in an experimental phase which means that there might be
```
Comment (lakshmi-kannan) [inline docs/source/actions.rst]: Please don't remove this. We aren't actively testing this runner at all.

Comment (humblearner) [inline docs/source/actions.rst]: ok

File: docs/source/config/webui.rst [hunk 2]
```diff
@@ -6,7 +6,7 @@ st2web is an Angular-based HTML5 web application. It allows you to control the w
 Deployment
 -----------
 
-Production version of st2web is deployed on all-in-one installation. You can access the UI by pointing your browser to ``https://<server hostname>/``. For vagrant deployment of st2express, it would be https://172.168.50.11/.
+Production version of st2web is deployed with the installation. You can access the UI by pointing your browser to ``https://<server hostname>/``. For vagrant deployment of st2express, it would be https://172.168.50.11/.
```
Comment (LindsayHill) [inline docs/source/config/webui.rst]: Haven't we deleted st2express? So this needs updating? (And probably a few other locations where we refer to st2express)

**Comment by humblearner** [comment]
First two changes are from https://github.com/StackStorm/st2docs/pull/184

**Comment by Kami** [comment]
LGTM.

Thanks.

**Pull request merged**